crates/bevy_pbr/src/atmosphere/resources.rs

```diff
@@ -185,19 +246,23 @@ impl FromWorld for RenderSkyBindGroupLayouts {
                     (4, uniform_buffer::<GpuLights>(true)),
```

Comment:
same here, ::sequential

Reply:
For the sequential, I've thought about cleaning this up, yet another separate, small PR